Packages/manifest.json

```diff
@@ -14,7 +14,8 @@
     "com.unity.visualscripting": "1.8.0",
     "eu.netherlands3d.obj-importer": "1.0.0",
     "eu.netherlands3d.transform-handles": "1.0.0",
-    "nl.netherlands3d": "https://github.com/Amsterdam/Netherlands3D.git?path=/Packages/Netherlands3D/",
+    "eu.netherlands3d.web-cursors": "1.0.2",
```

Comment:
Zie ik goed dat de web cursor package ook expliciet geinstalleerd is? Niet perse nodig als het al een dependency is in Amsterdam/Netherlands3D; sterker nog: als er expliciet een package met een specifiek versienummer geinstalleerd wordt kan dat bij-effecten hebben vanwege de manier waarop UPM versies resolved. Zie https://docs.unity3d.com/Manual/upm-dependencies.html voor meer informatie

Als het wel gewoon een dependency is mag je deze opmerking negeren; ik kan deze file nog niet 100% accuraat lezen 😓

Reply:
goeie, ik zal hem daar weghalen